src/ui/Card/Card.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card, CardProps, SecondaryCard } from ".";
+import type { Meta, StoryObj } from "@storybook/react";
+
+const CardComponent = (props: CardProps) => <Card {...props} />;
+const SecondaryCardComponent = (props: CardProps) => (
+  <div style={{ width: "400px" }}>
+    <SecondaryCard {...props} />
+  </div>
+);
+
+const meta: Meta<typeof Card> = {
+  component: CardComponent,
+  title: "ui/Card",
+  args: {
+    title: {
+      text: "Report",
+      iconName: "flag",
+    },
+    content: (
+      <div>
+        Banjo-Kazooie is the first game in the Banjo-Kazooie series. It was
+        developed by Rare and released in 1998 for the Nintendo 64. In this
+        game,
+      </div>
+    ),
+    footer: <footer>Footer content</footer>,
+  },
+  argTypes: {
+    theme: {
+      control: {
+        type: "inline-radio",
+        options: ["light", "dark"],
+      },
+      defaultValue: "light",
+    },
+  },
+};
+
+export const Default: StoryObj<typeof Card> = CardComponent.bind({});
```

Comment:
**suggestion**: This throws an error and follows the Component Story Format 2, which is the old format. To transform it to [CSF3](https://storybook.js.org/docs/react/api/csf#upgrading-from-csf-2-to-csf-3) you can do the following:

```javascript
const meta: Meta<typeof Card> = {
  component: Card,
  title: "ui/Card",
  args: {
    title: {
      text: "Report",
      iconName: "flag",
    },
    content: (
      <div>
        Banjo-Kazooie is the first game in the Banjo-Kazooie series. It was
        developed by Rare and released in 1998 for the Nintendo 64. In this
        game,
      </div>
    ),
    footer: <footer>Footer content</footer>,
  },
  argTypes: {
    theme: {
      control: {
        type: "inline-radio",
        options: ["light", "dark"],
      },
      defaultValue: "light",
    },
  },
};

export default meta;
type Story = StoryObj<typeof Card>;

export const Default: Story = {
  render: (args) => <Card {...args} />,
};

export const Secondary: Story = {
  args: {
    center: true,
    title: {
      text: "Kudometer",
    },
    date: "13th of May 1991",
    description: "This is a description",
  },
  render: (args) => (
    <div style={{ width: "400px" }}>
      <SecondaryCard {...args} />
    </div>
  ),
};
```

Reply:
Fixed!